From 3f9a7a39b01f6a536b19c2e684076430631156f2 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 19 Aug 2016 18:00:28 +0800
Subject: [PATCH 1132/5242] MLK-13096-02 cpufreq: imx: update the cpufeq for
 imx6ll

commit  dbba81b8e9bdbac40dc6529e252bebcf3388b256 from
https://source.codeaurora.org/external/imx/linux-imx.git

According the the latest datasheet, we updated the lowest
OPP to 198MHz. So we need to update the cpufreq code to fix
the syttem hang issue when run 'cpufreq-info' in low bus mode
on i.MX6ULL.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/cpufreq/imx6q-cpufreq.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/cpufreq/imx6q-cpufreq.c b/drivers/cpufreq/imx6q-cpufreq.c
index 9c67264..b009ec7 100644
--- a/drivers/cpufreq/imx6q-cpufreq.c
+++ b/drivers/cpufreq/imx6q-cpufreq.c
@@ -26,7 +26,7 @@
 #define FREQ_1P2_GHZ		1200000000
 #define FREQ_396_MHZ		396000
 #define FREQ_696_MHZ		696000
-#define FREQ_99_MHZ		99000
+#define FREQ_198_MHZ		198000
 #define FREQ_24_MHZ		24000
 
 static struct regulator *arm_reg;
@@ -89,10 +89,10 @@ static int imx6q_set_target(struct cpufreq_policy *policy, unsigned int index)
 	/*
 	 * ON i.MX6ULL, the 24MHz setpoint is not seen by cpufreq
 	 * so we neet to prevent the cpufreq change frequency
-	 * from 24MHz to 99Mhz directly. busfreq will handle this
+	 * from 24MHz to 198Mhz directly. busfreq will handle this
 	 * when exit from low bus mode.
 	 */
-	if (old_freq == FREQ_24_MHZ && new_freq == FREQ_99_MHZ) {
+	if (old_freq == FREQ_24_MHZ && new_freq == FREQ_198_MHZ) {
 		return 0;
 	};
 
-- 
1.7.9.5

